### ISSUE TITLE
Survival in days should be context aware

### ISSUE THREAD

**Issue by jmcmurry** [TheJacksonLaboratory/PhenopacketLab#147]
<img width="1447" alt="Screenshot 2023-03-14 at 1 51 38 PM" src="https://user-images.githubusercontent.com/3658863/225133618-5f92b0d5-e76b-43ca-b028-ad100bfdd066.png">

It would be better to have survival that option only show up if the person is deceased. It might also be appropriate to capture time in more flexible units. Days is definitely standard unit for NICU. Someone who passes away at the age of 14 years would be a level of precision in days that is neither appropriate nor useful. This is a low priority fix for now, but not too difficult either.

**Comment by belkassaby** [comment]
So is survival time the age the individual was deceased? I thought that it was related the number of days a patient was alive after diagnosis (that's what the documentation says: https://phenopacket-schema.readthedocs.io/en/latest/vital-status.html#rstvitalstatus). I will make it appear only if deceased if selected.

**Comment by belkassaby** [comment]
here is a screenshot of how the fields are hidden if not deceased status: 
![Screen Shot 2023-03-16 at 10 09 11 AM](https://user-images.githubusercontent.com/1378819/225643706-bba82f51-d520-4774-a2a6-9f87fc6d03ad.png)
![Screen Shot 2023-03-16 at 10 09 21 AM](https://user-images.githubusercontent.com/1378819/225643719-19715a13-8946-4f13-a3c8-e7d711b379a6.png)

**Comment by jmcmurry** [comment]
Ahhh! I thought you were doing this for nicu situation. Survival time in days post cancer treatment seems rather specific. Could we either a) remove this altogether for now, b) prompt for this only after a cancer treatment is entered in MAxO step?

**Comment by BethSundberg** [comment]
tested and it looks good.
